boards/nxp/frdm_mcxa156/frdm_mcxa156.yaml

```diff
@@ -16,4 +16,5 @@ toolchain:
   - xtools
 supported:
   - gpio
+  - cmp
```

Comment:
Is this test tag used anywhere in Zephyr? I don't see any yaml files referencing it.